### ISSUE TITLE
Set role="alert" as default for snackbar

### ISSUE THREAD

**Issue by szhang-eightfold** [EightfoldAI/octuple#503]
Currently from my observation, Snackbar component uses InfoBar which has default role="presentation", however it should have role="alert" by default for Snackbar as it will help with a11y users to announce the alert message.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role#:~:text=The%20alert%20role%20is%20used,can%20then%20notify%20the%20user.

Let me know if this is not the case though.

**Comment by dkilgore-eightfold** [comment]
Tracking change with ENG-41361

**Comment by dkilgore-eightfold** [comment]
Will update the storybook prop in a future change, component has been updated.
